tests/e2e/backup_restore_suite_test.go

```diff
@@ -29,6 +29,14 @@ var _ = Describe("AWS backup restore tests", func() {
 
 		err = vel.CreateOrUpdate(&vel.CustomResource.Spec)
 		Expect(err).NotTo(HaveOccurred())
+
+		log.Printf("Waiting for velero pod to be running")
+		Eventually(isVeleroPodRunning(namespace), time.Minute*3, time.Second*5).Should(BeTrue())
```

Comment:
This is awesome.. is it possible to kick the time vars out to a config file in a future revision?  In the case one environment is faster/slower than others? ++   THANK YOU!!!